panther_analysis_tool/main.py

```diff
@@ -810,6 +829,67 @@ def setup_parser() -> argparse.ArgumentParser:
     return parser
 
 
+def zip_managed_schemas(args: argparse.Namespace) -> Tuple[int, str]:
+    """Packs managed schemas of a tagged release into a local zip file.
+
+    Args:
+        args: The populated Argparse namespace with parsed command-line arguments.
+
+    Returns:
+        A tuple of return code and the archive filename.
+    """
+
+    manifest = []
+    with tempfile.TemporaryDirectory(prefix="zip-managed-schemas-") as tmp_dir:
+        repo_url = "https://github.com/panther-labs/panther-analysis"
+        repo_dir = os.path.join(tmp_dir, "panther-analysis")
+        rel = args.release
+        if not semver.VersionInfo.isvalid(
+                rel[1:] if rel.startswith("v") else rel):
+            logging.error("Invalid release tag %s", rel)
+            return 1, ""
+
+        logging.info("Cloning %s tag of %s", rel, repo_url)
+        # nosec
+        cmd = [
+            "git", "clone", "--branch", rel, "--depth", "1", "-c",
+            "advice.detachedHead=false", repo_url, repo_dir
+        ]
+        result = subprocess.run(cmd, check=True, timeout=120)  # nosec
```

Comment:
`check=True` will raise an exception on non-zero exit codes, so the check below is not necessary, or better phrased this code branch will never run. Was [this](https://bandit.readthedocs.io/en/latest/_modules/bandit/plugins/injection_shell.html) the check you want to bypass?